cypress/integration/e2e/application_spec.js

```diff
@@ -60,13 +75,26 @@ describe("Application Spec", () => {
     cy.intercept("/api/pattern/deploy*").as("applicationDeploy")
     cy.get('[data-cy="deploy-btn-confirm"]').click();
     cy.wait("@applicationDeploy").then(() => {
-      // cy.get("[data-cy='progress-snackbar']").contains("Deploying design");
+      // cy.get("[data-cy='progress-snackbar']").contains(`Deploying application: ${bookInfoApp}`);
       cy.get("body").then(body => {
         if (body.find("[aria-describedby='notistack-snackbar'] #notistack-snackbar").length > 0) {
           cy.get("[aria-describedby='notistack-snackbar'] #notistack-snackbar").should("not.contain", "Error")
         }
       })
     })
+    //Undeploy 
+    cy.get('#undeploy-design-btn').click();
+    // modal opens
+    cy.intercept("/api/pattern/deploy*").as("applicationUndeploy")
+    cy.get('[data-cy="deploy-btn-confirm"]').click();
+    // cy.get("[data-cy='progress-snackbar']").contains(`Undeploying application: ${bookInfoApp}`);
+    cy.wait("@applicationUndeploy").then(() => {
+      cy.get("body").then(body => {
+        if (body.find("[aria-describedby='notistack-snackbar'] #notistack-snackbar").length > 0) {
+         cy.get("[aria-describedby='notistack-snackbar'] #notistack-snackbar").should("not.contain", "Error")
```

Comment:
Let's change it to "Error" or ""Failed" in checking... because I see when there is error, it says deploy "failed"

Reply:
is it getting difficult, @Chadha93 ?

Reply:
Will update it by tomorrow.